src/resmom/mom_main.c

```diff
@@ -6872,7 +6892,7 @@ job 	*pjob;
  *
  */
 int
-mom_over_limit(job *pjob)
+mom_over_limit(job *pjob, int enforce_job_wide)
```

Comment:
Please change the function header to reflect this new argument

Reply:
header changed